test/libyul/yulOptimizerTests/blockFlattener/basic.yul

```diff
@@ -8,8 +8,9 @@
   }
   let z := mload(2)
 }
+// ====
+// step: blockFlattener
```

Comment:
Could we call it `optimizerStep` ?

Reply:
I would say that settings should either be generic or specific to the test suite. Why the longer name?

Reply:
Did not realize that the settings are local to each suite.

Reply:
Each of them can pick and choose which settings it wants to react to.